src/Components/FinanceDataInput/FinanaceDataInput.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+import { useState, useEffect } from 'react';
+import './FinanceDataInput.scss';
+
+const FinanceDataInput = (prop: any) => {
+    const [value, setValue] = useState('');
+    const [error, setErrorState] = useState (false);
+    const labelErrorMessage = document.getElementById('label');
+
+    
+
+    const handleChange= (e: any) => {
+        setValue(e.target.value);
+        if (/[^a-zA-Z\s]/.test(e.target.value)){
+            setErrorState(false);
+        }
+        else {
+            setErrorState(true);
+        }
+    }
+
+    useEffect(() => {
+        if (labelErrorMessage !== null && error === true){
+            labelErrorMessage.innerHTML='This field is required.'
```

Comment:
suggestion: Using a react `useState` object is better for React in general as it gives you more control over updating the component. This is still a viable option for this use case but the component won't re-render so there may be weird bugs caused by this.